node/external/transactionAPI/apiTransactionProcessor.go

```diff
@@ -515,6 +510,7 @@ func (atp *apiTransactionProcessor) selectTransactions(accountsAdapter state.Acc
 		return nil, err
 	}
 
+	// TODO use the right information for blockchainInfo
```

Comment:
:+1: 

(though, if missing, it means "derive the virtual session using all tracked blocks", which is perfectly fine)